src/api/app/helpers/webui/group_helper.rb

```diff
@@ -6,4 +6,9 @@ def group_management_label
       'Group Management'
     end
   end
+
+  def group_with_icon(group_title)
+    group = Group.find_by(title: group_title)
+    user_image_tag(group, alt: group_title) + ' ' + link_to(group_title, group_show_path(group))
```

Comment:
Nitpick: IIRC you could write this as
```ruby
link_to() do
  user_image_tag
```

Reply:
and I would consider using [image_tag_for](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/helpers/webui/webui_helper.rb#L421) since we use it for the other icons as well

Reply:
This is the old UI not the new one. :wink: and the link is only in the name